File: tests/cpp/IntensiveCouplingTest/NotIntensiveCouplingTest1.cpp
// The class has CINT larger than 8 but max nesting lower than 2.
// -> Has no intensive coupling smell.
#include "NotIntensiveCouplingTest1.h"
int NotIntensiveCouplingTest1 :: func1N1(){
    int v1 = this->func2N1();
    int v2 = this->func3N1();
    int v3 = this->func4N1();
    int v4 = this->func5N1();
    int v5 = this->func6N1();
    int v6 = this->func7N1();
    int v7 = this->func8N1();
    int v8 = this->func9N1();
    int v9 = this->func10N1();
    for (int i=0; i < 10; i++){
            v1 --;
    }
}
int NotIntensiveCouplingTest1 :: func2N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func3N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func4N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func5N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func6N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func7N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func8N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func9N1(){
    return 0;
}
int NotIntensiveCouplingTest1 :: func10N1(){
    return 0;
}

File: tests/java/IntensiveCouplingTest/NotIntensiveCouplingTest1.java
// The class has CINT larger than 8 but max nesting lower than 2.
// -> Has no intensive coupling smell.


public class NotIntensiveCouplingTest1 //Type originates from: NotIntensiveCouplingTest1.h
{
	private int f1;
	private int f2;
	private int func1() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		int v1 = this.func2();
		int v2 = this.func3();
		int v3 = this.func4();
		int v4 = this.func5();
		int v5 = this.func6();
		int v6 = this.func7();
		int v7 = this.func8();
		int v8 = this.func9();
		int v9 = this.func10();
		for (int i = 0; i < 10; i++)
		{
				v1--;
		}
	}

	private int func2() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func3() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func4() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func5() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func6() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func7() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func8() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func9() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}

	private int func10() //Method definition originates from: NotIntensiveCouplingTest1.cpp
	{
		return 0;
	}
}
